src/test/java/uk/gov/justice/digital/delius/controller/secure/AttendanceResourceAPITest.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import static io.restassured.module.mockmvc.RestAssuredMockMvc.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import io.restassured.module.mockmvc.RestAssuredMockMvc;
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import uk.gov.justice.digital.delius.controller.advice.SecureControllerAdvice;
+import uk.gov.justice.digital.delius.data.api.Attendance;
+import uk.gov.justice.digital.delius.data.api.Attendances;
+import uk.gov.justice.digital.delius.jpa.standard.entity.Contact;
+import uk.gov.justice.digital.delius.service.AttendanceService;
+import uk.gov.justice.digital.delius.service.AttendanceServiceTest;
+
+@SuppressWarnings("OptionalGetWithoutIsPresent")
+@RunWith(MockitoJUnitRunner.class)
+public class AttendanceResourceAPITest {
```

Comment:
Very minor but we have settled on tests that are end-to-end (use the database) to be named with the word API in it and ones that mock out the service without - so this would be named `AttendanceResourceTest`

Reply:
Done